src/PropertyBuilder/OptionBuilder.php

```diff
@@ -46,7 +46,7 @@ private function resolveProductOptions(ProductInterface $product, Document $docu
                     'code' => $productOptionValue->getOption()->getCode(),
                     'translations' => $translations,
                     'value' => [],
-                    'onHand' => $productVariant->getOnHand(),
+                    'onHand' => $productVariant->isTracked() ? 1 : $productVariant->getOnHand() - $productVariant->getOnHold(),
```

Comment:
I'm out of context, but this will not work I believe... 
OnHand/OnHold have sense only on products with `isTracked()===true`, so correct logic is a bit different I guess:

```
'onHand' => $productVariant->isTracked() ? $productVariant->getOnHand() - $productVariant->getOnHold() : 1,
```

Reply:
Looks right, Igor